sockeye/utils.py

```diff
@@ -667,6 +675,31 @@ def all_gather_object(obj: T) -> List[T]:
     return obj_list
 
 
+# Track whether DeepSpeed has been initialized
+_using_deepspeed = False
+
+def init_deepspeed():
+    """
+    Make sure all of the DeepSpeed modules we use can be imported, initialize
+    DeepSpeed, and set the global variable that tracks initialization.
+
+    """
+    global _using_deepspeed
+    try:
+        import deepspeed  # pylint: disable=E0401
+        import deepspeed.utils.zero_to_fp32  # pylint: disable=E0401
+        deepspeed.init_distributed()
```

Comment:
Would it make sense to add a logging statement here?

Reply:
We get free log messages here from [`init_distributed()`](https://github.com/microsoft/DeepSpeed/blob/b288cf1b9b90030cc2025f2681e77f3bc81dd409/deepspeed/comm/comm.py#L577).